foxy/api/action_msgs/msg/GoalStatus.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <title>msg/GoalStatus Documentation</title>
+    <link type="text/css" rel="stylesheet" href="../../msg-styles.css" />
+  </head>
+  <body>
+    <div id="container">
+      <h1 class="msg-title"><a href="../index-msg.html">action_msgs</a>/msg/GoalStatus Message</h1>
+      <div class="msg">
+        <h2>File: <span class="filename">action_msgs/msg/GoalStatus.msg</span></h2>
+        <h2>Raw Message Definition</h2>
+        <div class="raw-msg">
+          <a style="color:blue"># An action goal can be in one of these states after it is accepted by an action server.</a><br>
+          <a style="color:blue"># For more information, see http://design.ros2.org/articles/actions.html</a><br>
+          int8 STATUS_UNKNOWN   = 0<br>
```

Comment:
Not sure about using <p>, this tag is adding a new line, right?

Reply:
True, but one of the advantages of using a `<p>` element is that it can have an `id` attribute or a `class` attribute, so you could do fancier styling by modifying the .css file.